doc/requirements.txt

```diff
@@ -1 +1,2 @@
 signac
+jsonschema
```

Comment:
This file can be deleted, it's missing several other dependencies. I updated the ReadTheDocs config to use the root `requirements.txt`.

Reply:
@bdice RTD is failing because we need to pin docutils. Where would you like to put that pinning since we don't currently bother to pin anything docs related explicitly?

Reply:
I'll fix this upstream. We just need to use a newer RTD build environment.

Reply:
I'll delete this file and merge once I get the RTD config fixed.

Reply:
Just kidding. I'll change it on this branch so we know it works.